### PR TITLE
Create and implement theme

### PR DESCRIPTION
**Pull request by EraAmate**

### CODE REVIEW

File: client/src/GlobalStyles.js [hunk 1]
```diff
@@ -9,16 +9,15 @@ function GlobalStyles() {
...
         body {
           margin: 0;
           height: 100vh;
-          font-family: 'Nadeem Regular', regular;
+          font-family: 'Montserrat', sans-serif;
           font-size: 16px;
-          color: #010101;
-          display: flex;
-          justify-content: center;
+          margin: 0px;
```
Comment (SebastianStupp) [inline client/src/GlobalStyles.js:19]: Margin seems to be twice in here, maybe u can remove one :)

File: client/.storybook/themeDecorator.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { ThemeProvider } from 'emotion-theming';
+import GlobalStyles from '../src/GlobalStyles';
+import theme from '../src/theme';
+
+const ThemeDecorator = (storyFn) => (
+  <ThemeProvider theme={theme}>
+    <GlobalStyles />
```
Comment (marwinburesch) [inline client/.storybook/themeDecorator.js:8]: `GlobalStyles` doesn't belong here, please move it to `client/.storybook/config.js`, you can implement it just like `themeDecorator`.

File: client/src/theme.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    linearTertiary:
+      'linear-gradient(to bottom, #d4ad7a 0%, #c2a074 50%, #ab906d 100%)',
+    shadow: '#00000029',
+    borderPrimary: '#AB906D',
+    borderSecondary: '#E5DDD2',
+    area: '#F2F2F2',
+    attention: 'red',
+    available: 'green',
+  },
+};
```
Comment (marwinburesch) [inline client/src/theme.js:19]: Just an idea of how theming could work as well, also there was a small typo in `linearPrimary` 
```suggestion
const theme = {
  colors: {
    basic: '#FFFFFF',
    textPrimary: '#141414',
    textSecondary: '#FFE0A6',
    shadow: '#00000029',
    borderPrimary: '#AB906D',
    borderSecondary: '#E5DDD2',
    area: '#F2F2F2',
    attention: 'red',
    available: 'green',
  },
  gradients: {
   linearPrimary:
      'linear-gradient(to bottom, #D5AE7A 0%, #917F66 50%, #575656 100%)',
   linearSecondary:
      'linear-gradient(to bottom, #fcf1da 0%, #d2c5aa 70%, #a79979 100%)',
   linearTertiary:
      'linear-gradient(to bottom, #d4ad7a 0%, #c2a074 50%, #ab906d 100%)'
   }
};
```

File: client/.storybook/config.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+import { configure, addDecorator } from '@storybook/react';
```
Comment (marwinburesch) [inline client/.storybook/config.js:1]: DeepScan fails because of this
```suggestion
import { addDecorator } from '@storybook/react';
```

**Pull request merged**